src/main/java/net/imagej/ops/filter/FilterNamespace.java

```diff
@@ -30,8 +30,11 @@
 
 package net.imagej.ops.filter;
 
+import java.util.Arrays;
 import java.util.List;
 
+import org.scijava.plugin.Plugin;
```

Comment:
According to [ImageJ code style](https://imagej.net/Coding_style#Eclipse_code_style_profiles), the imports should be ordered strictly alphabetical.

Reply:
I am using imglib2 formatter. Is there an ImageJ formatter?

Reply:
Yes: https://github.com/imagej/imagej/tree/master/config

Reply:
Noted.